### ISSUE TITLE
Drag across multiple packery containers

### ISSUE THREAD

**Issue by newtriks** [metafizzy/packery#27]
Hi,

Thanks for a fantastic resource, it's a great library without a doubt. One thing I was looking at doing was dragging items across multiple packery containers. Would this be possible or are there any particular blockers you would expect? 

Would you have a working example available if you see this as a possibility?

Cheers,

Simon

**Comment by rameshsrinivasan** [comment]
+1

**Comment by atteeela** [comment]
+!

**Comment by noxoc** [comment]
+1, any update on this?

**Comment by dnascimento** [comment]
+1

**Comment by glenpike** [comment]
+1

We'd like to be able to add instances of Draggabilly items (not necessarily from another Packery container).

**Comment by rvetere** [comment]
+1

**Comment by orenqi** [comment]
This is my walk around:
'unbindDraggabillyEvents' is added to unbind event from old container

	function dragItemToPage(draggie, id_from, id_to) {
  		// Move item to new container
  		$(draggie.element).detach().appendTo(id_to);
  		$(id_from).packery( 'reloadItems' );
  		$(id_to).packery( 'reloadItems' );

  		// Rebind draggie events
		$(id_from).packery( 'unbindDraggabillyEvents', draggie );
		$(id_to).packery( 'bindDraggabillyEvents', draggie );
		$(id_to).packery( 'stamp', draggie.element );
		var item = $(id_to).packery( 'getItem', draggie.element );
		if ( item ) {
			item.dragStart();
		}
	}

**Comment by dominicshaw** [comment]
+1

**Comment by JesseR5095** [comment]
This is a pretty simple problem. I see no reason it needs to be integrated in Packery core. Simply track mouse position and check to see if the mouse collides with a new Packery container when the drag event ends. If true, then remove the event.target from the original container, inject its contents into a new div with the same class and id, and append it to the new container.

**Comment by BenRacicot** [comment]
+1 !!!
Would it be possible to just create a packery item instance on specific mousedown/click/touchstart to replicate a grab? Then "dragging" this selection across the packery area would behave as usual packery drag behavior...

**Comment by lerzenit** [comment]
+1

**Comment by niconil** [comment]
+1

**Comment by timothylippiatt** [comment]
+1

**Comment by AbelToy** [comment]
+1

**Comment by antenando** [comment]
+1
![](http://i.giphy.com/l2R032V7qRAF8J6qA.gif)

**Comment by NewLIST** [comment]
+100, ah, WHY hasn't this been implemented yet?!?

**Comment by reubenpressman** [comment]
Really.. over 2 years with nothing on whether this is going to happen or not?

**Comment by BobWalsh** [comment]
+1

**Comment by lunarose23** [comment]
+1

**Comment by KyleKaniecki** [comment]
+1

**Comment by jpotth** [comment]
+1

**Comment by brianputz** [comment]
+1, coming up on four years with this open, is it safe to assume this won't be done?

**Comment by krisztianodor** [comment]
+1

**Comment by desandro** [comment]
Thanks all for your input. This is clearly a hotly requested feature 🔥. I've locked the issue to limit the +1 comments. If I decide to add this feature to Packery, I'll be sure to follow up here.
